Solutions/Tanium/Analytic Rules/TaniumThreatResponseAlerts.yaml

```diff
@@ -0,0 +1,34 @@
+id: dd9aa0ff-7ac1-4448-879c-e1a18d5890b4
+name: Tanium Threat Response Alerts
+description: 'Alerts from Tanium Thread Response (THR) that can be acted upon by Tanium Sentinel Playbook'
+severity: High
```

Comment:
Please also add the requiredDataConnectors field.

Reply:
@ajcollins0 Please put the field here and leave it blank. Similar to what you have done for tactics.

Reply:
added - thank you!